packages/kit/src/exports/vite/graph_analysis/index.js

```diff
@@ -2,7 +2,9 @@ import path from 'node:path';
 import { posixify } from '../../../utils/filesystem.js';
 import { strip_virtual_prefix } from '../utils.js';
 import { app_server, env_dynamic_private, env_static_private } from '../module_ids.js';
+import { load_config } from '../../../core/config/index.js';
 
+const svelte_config = await load_config();
```

Comment:
loading the config again could be a breaking change in case a config contains one time setup code that is then executed twice.

Reply:
Yeah mentioned it above, didnt saw another way to get the config on this part of code

Reply:
that would have to be found then. using top-level-await here is also not great.